agentstack/cli/cli.py

```diff
@@ -20,86 +20,100 @@
 from .. import generation
 from ..utils import open_json_file, term_color, is_snake_case
 
+created_files = []
+created_dirs = []
 
-def init_project_builder(slug_name: Optional[str] = None, template: Optional[str] = None, use_wizard: bool = False):
-    if slug_name and not is_snake_case(slug_name):
-        print(term_color("Project name must be snake case", 'red'))
-        return
+def rollback_actions():
+    for file in created_files:
+        if os.path.exists(file):
+            os.remove(file)
+    for dir in created_dirs:
+        if os.path.exists(dir):
+            shutil.rmtree(dir)
```

Comment:
We should be very careful about deleting files when any exception occurs. Possible that running this command on an already initialized project would permanently erase user data.